app/templates/index.hbs

```diff
@@ -5,12 +5,59 @@
             <div class="row">{{!HEADER ROW}}
                 {{!TITLE}}
                 <div class="text-center m-t-lg col-xs-12">
-                    <div class="reviews-brand"></div>
-                    <p class="lead">
-                    </p>
+                    <div class="p-b-xl reviews-brand"></div>
                 </div>
             </div> {{!END ROW}}
-
         </div> {{!END CONTAINER}}
     </div> {{!END HEADER}}
+    <div class="reviews-middle"> {{!MIDDLE}}
+        <div class="container"> {{!CONTAINER}}
+            <div class="row col-md-6">
+                <div class="reviews-middle-features"></div>
+            </div>
+            <div class="col-md-6">
+                <h2>Moderate Your Collection</h2>
```

Comment:
@aaxelb thanks for your feedback, I will address your requests. The navbar is using the OSF styling we imported. It is same as one in preprints and used from ember-osf. There is another version of nav-bar used for preprint providers page. We are using the same version of ember-osf.